packages/generator/README.md

```diff
@@ -9,8 +15,16 @@ Then install this generator:
 `yarn global add @kaizen/generator-create`
 
 ## Usage
-`yo @kaizen/create:draft` 
+1. Run the following and follow the prompts: `yo @kaizen/create:draft` 
+2. Choose an appropriate name for your component. See our guide: https://cultureamp.design/components/overview/#naming-things-is-hard!
+3. Select whether you'd like to generate React component, Elm component (or both)
+4. Done 🎉
+   
+Run `yarn install` to add this new component to your workspace root. This is required before running `yarn storybook`. 
 
 ## Help 
 Q: I get an error when I run `yo` in a terminal
-A: Yarn bin might not be added to your PATH. Add this to `~/.bash_profile` (or your equivalent): `export PATH="$PATH:$(yarn global bin)"`
\ No newline at end of file
+A: Yarn bin might not be added to your PATH. Add this to `~/.bash_profile` (or your equivalent): `export PATH="$PATH:$(yarn global bin)"`
+
+Q: asdf is complaining about things
+A: `asdf reshim nodejs`
```

Comment:
😬 Sorry @didoesdigital I didn't know what error you were getting so I improvised.